### ISSUE TITLE
running module:make:request on a module with compound name

### ISSUE THREAD

**Issue by dlaynes** [caffeinated/modules#97]
Hi. Thanks for creating this library. Excuse my English.

When running this command:
````
php artisan module:make:request user-manager ProcessUser
````
The module name is converted to camel case format (**UserManager**) automatically.
But the method *Caffeinated\Modules\Repositories\Repository\ModuleRepository::exists()* is expecting an slug, not the default Camel Case name, resulting in the error "Module ModuleName does not exist".

The following does not work either:
````
php artisan module:make:request UserManager ProcessUser
````

So, I tried to hardcode the original slug in the method call, and I could create the new Request without problems.
Regards.

**Comment by kaidesu** [comment]
Thanks for the report, I'll be sure to get this patched soon~

**Comment by kaidesu** [comment]
Fixed with b582b79421929f3958c75d890ae5ffe8b7191141. I'll be included in the next tagged release of the package~
